vpr/src/route/router_lookahead_map_utils.h

```diff
@@ -25,8 +25,49 @@
 #include "rr_node.h"
 #include "rr_graph_view.h"
 
+/* we will profile delay/congestion using this many tracks for each wire type */
+#define MAX_TRACK_OFFSET 16
+
 namespace util {
 
+class Cost_Entry;
```

Comment:
Doxygen comments for these could be useful too.

Reply:
Will do doxygen commenting on lookahead utils in another PR.